src/main/kotlin/no/nav/syfo/domain/SenOppfolgingStatus.kt

```diff
@@ -2,5 +2,13 @@ package no.nav.syfo.domain
 
 enum class SenOppfolgingStatus(val isActive: Boolean) {
     KANDIDAT(isActive = true),
-    FERDIGBEHANDLET(isActive = false),
+    FERDIGBEHANDLET(isActive = false), ;
+
+    companion object {
+        fun from(type: VurderingType): SenOppfolgingStatus {
+            when (type) {
+                VurderingType.FERDIGBEHANDLET -> return FERDIGBEHANDLET
+            }
+        }
+    }
```

Comment:
Kandidaten har en status, men vurderingen er av en type? Er det dumt at vi har to konsepter? Kafkatopicet heter det også `-status`.

Reply:
Tenkte det ga mening å skille de siden en vurdering ikke kan ha status/type `KANDIDAT` :thinking: Kunne sikkert uttrykt det på andre måter også, men føler det blir mest riktig at en kandidat har status og vurdering er av en viss type...

Reply:
Ja, det er godt poeng 👍🏼